Library/Formula/antlr.rb

```diff
@@ -13,4 +11,23 @@ def install
       java -classpath #{prefix}/antlr-#{version}-complete.jar:. org.antlr.v4.runtime.misc.TestRig "$@"
     EOS
   end
+
+  test do
+    path = testpath/"Expr.g4"
+    path.write <<-EOS.undent
+    grammar Expr;
+    prog:\t(expr NEWLINE)* ;
+    expr:\texpr ('*'|'/') expr
+        |\texpr ('+'|'-') expr
+        |\tINT
+        |\t'(' expr ')'
+        ;
+    NEWLINE :\t[\\r\\n]+ ;
+    INT     :\t[0-9]+ ;
+    EOS
+    ENV["CLASSPATH"] = ".:#{prefix}/antlr-#{version}-complete.jar:#{ENV["CLASSPATH"]}"
```

Comment:
@jacknagel This doesn’t work because [`ENV.prepend_path`][pp] uses a space as a separator, while here it should use a colon (`:`). The following works:

```rb
ENV.prepend "CLASSPATH", "#{prefix}/antlr-#{version}-complete.jar", ":"
ENV.prepend "CLASSPATH", ".", ":"
```

I edited the pull request with this new syntax.

[pp]: https://github.com/Homebrew/homebrew/blob/71138f7bf9398afe94fe9051a665c14324929f6a/Library/Homebrew/extend/ENV/shared.rb#L81